lib/views/list-item/index.js

```diff
@@ -141,13 +141,13 @@ class ListItem extends React.Component {
         return self.isStatic() || !self.isExpanded();
       },
 
-      onResponderGrant({nativeEvent}) {
-        self.tapStart = getTime(nativeEvent);
-        debug('on responser grant', nativeEvent);
+      onResponderGrant() {
+        self.tapStart = Date.now();
+        debug('on responser grant');
       },
 
-      onResponderRelease({nativeEvent}) {
-        var time = getTime(nativeEvent) - self.tapStart;
+      onResponderRelease() {
+        var time = Date.now() - self.tapStart;
```

Comment:
For some reasons the timestamps we were getting from the `nativeEvent` objects we're incorrect. `Date.now()` seemed to work more reliably.